docs/docs/examples/elasticsearch/BITBUCKET_ELASTICSEARCH.md

```diff
@@ -12,28 +12,40 @@ helm repo add elastic https://helm.elastic.co
 ```
 then install it:
 ```bash
-helm install elasticsearch --version 7.9.3 elastic/elasticsearch
+helm install elasticsearch --set image.tag="7.9.3" elastic/elasticsearch
```

Comment:
The standard `version` helm install parameter doesn't work for elasticsearch chart?

Reply:
No... It was causing error during the blitz. I think `version` is the helm chart version, while `image.tag` is the docker image version for Elasticsearch.